cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -625,6 +625,12 @@ body {
             }
 
             &-list {
+                .grid_column(3);
+            }
+
```

Comment:
TRIVIAL: Remove extra whitespace here, should be only one newline.

Reply:
Removed in https://github.com/cfpb/cfgov-refresh/pull/3396/commits/7bdc211bb2b751c741a20ea9a524bf9aeb56c863